src/timer/Timer.js

```diff
@@ -1,17 +1,18 @@
-function Timer(seconds) {
-  this.seconds = seconds;
-}
+class Timer {
+  constructor(seconds) {
+    this.seconds = seconds;
+  }
 
-Timer.prototype.start = function() {
-  var instance = this;
-  var timerInterval = setInterval(function() {
-    if (instance.seconds === 0) {
-      clearInterval(timerInterval);
-    }
+  start() {
+    this.timerInterval = setInterval(() => {
+      if (this.seconds === 0) {
+        clearInterval(timerInterval);
```

Comment:
You defined timerInterval with this.timerInterval, so on this line it will evaluate to undefined since you did not include 'this' keyword